jwala-webservices/src/main/java/com/cerner/jwala/ws/rest/v1/service/media/impl/MediaServiceRestImpl.java

```diff
@@ -71,9 +72,22 @@ public Response updateMedia(final JpaMedia media, final AuthenticatedUser aUser)
     }
 
     @Override
-    public Response removeMedia(final String name, final AuthenticatedUser aUser) {
+    public Response removeMedia(final String name, String type, final AuthenticatedUser aUser) {
+        MediaType mediaType = null;
+        if (type == null) {
```

Comment:
SInce type is a required PathParam I would throw an exception here. In general we don't want to set defaults for parameters that aren't provided or don't match our enums.